common/src/stack/switch/pylib/switch/m7800.py

```diff
@@ -67,6 +68,78 @@ def connect(self):
 
 		self._api_connection = mellanoknok.Mellanoknok(self.switch_ip_address, password=self.password)
 
```

Comment:
Move these functions to the end of the file, most people looking at this code will be looking for the connection stuff, followed by the IB partitioning code...